test/configuration/tests_params.json

```
@@ -1238,7 +1238,56 @@
                         },
                         "ground_truth_validation":
                         {
-                            "file_path": "<CONFIG_RELATIVE_PATH>/test_solutions/test_34_connect_locally_with_remote_server.snapshot"
+                            "file_path": "<CONFIG_RELATIVE_PATH>/test_solutions/test_32_fds_two_connected_servers_with_clients.snapshot"
+                        }
+                    }
+                },
+                "fastddstool_1":
+                {
+                    "kill_time": 5,
+                    "tool_config":
+                    {
+                        "id" : 0,
+                        "address": "127.0.0.1",
+                        "port": 34811
```

```suggestion
                        "port": 32811
```
Test 32, use 32k port instead.

---

Test renamed to 35 in 4d558d0